protected/views/site/help.php

```diff
@@ -4,16 +4,17 @@
 
 <div class="content">
             <div class="container">
-                <section class="page-title-section">
+                <div class="section page-title-section">
                     <div class="page-title">
-                        <ol class="breadcrumb pull-right">
-                            <li><a href="/">Home</a></li>
-                            <li><a href="about">About</a></li>
```

Comment:
I did not understand why a link to the about page is included in the breadcrumbs so I decided to remove it